tests/h/views/badge_test.py

```diff
@@ -11,10 +11,12 @@
 
 
 class TestBlocklist:
-    @pytest.mark.parametrize("bad_part", Blocklist.BLOCKED_DOMAINS)
+    @pytest.mark.parametrize("domain", Blocklist.BLOCKED_DOMAINS)
     @pytest.mark.parametrize("prefix", ("http://", "https://", "httpx://", "//"))
-    def test_it_blocks(self, bad_part, prefix):
-        url = f"{prefix}{bad_part}/path?a=b"
+    @pytest.mark.parametrize("suffix", ("", "/", "/path?a=b"))
+    def test_it_blocks(self, domain, prefix, suffix):
+        url = f"{prefix}{domain}{suffix}"
+        print(url)
```

Comment:
There is a `print` line that got left in the code by mistake.

Reply:
Gone now. Thanks